### PR TITLE
Fix and update the typescript template

### PR DESCRIPTION
**Pull request by NicolasMahe**
I had some issue when developing a new service with typescript.

- `npm ci` didn't work because the typescript compilation requires lots of dev dependencies
- add `tslint` and `@types/node` to dependencies
- add `"esModuleInterop": true` to tsconfig.json

### CODE REVIEW

**Comment by antho1404** [comment]
- `npm ci` to `npm i` is great
- `esModuleInterop` is great

- `tslint` is not needed. Needed if you want to lint but that's included in this template 
- `@types/node` as well should not be needed except if your application needs it

Why did you add these 2 dependencies? They should not be necessary

**Comment by NicolasMahe** [comment]
> * tslint

I think I saw it was the good practice to have `tsc`, `typescript` and `tslint` as dev dep.

> * @types/node

Because every typescript app I created with node had some weird unknown type without this dep

File: packages/cli/assets/services/typescript/package.json [hunk 1]
```diff
@@ -7,7 +7,9 @@
     "@liteflow/service": "^1.0.0-alpha.2"
```
Comment (antho1404) [inline packages/cli/assets/services/typescript/package.json:7]:
```suggestion
    "@liteflow/service": "^1.0.0"
```

File: packages/cli/assets/services/typescript/package.json [hunk 2]
```diff
@@ -7,7 +7,9 @@
...
   },
   "devDependencies": {
+    "@types/node": "^13.13.5",
     "tsc": "^1.20150623.0",
+    "tslint": "^6.1.2",
```
Comment (antho1404) [inline packages/cli/assets/services/typescript/package.json:12]:
```suggestion
```

**Comment by antho1404** [comment]
I don't want to enforce any kind of lint, if we add lint it means we need to have a default lint config that will make some people unhappy and some happy so I think better to keep it with "bring your own lint" and it's not something we add.

for the `@types/node`, this is again if your application needs it which might not be all of them. I'm ok to have it as yes it is quite common and really quickly needed.

I would just revert the linter

**Pull request merged**